index.html

```diff
@@ -2,22 +2,22 @@
   <head>
     <!--TODO Something in this meta tag causes a thin white line to appear to the right in iPhone5 only - ok on all other iPhones: http://www.kylejlarson.com/blog/2012/iphone-5-web-design/ -->
     <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
-        
+
     <link rel="stylesheet" href="./main.css" type='text/css'>
-    
+
     <!--FontAwesome-->
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
     <!-- Adding google fonts -->
     <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300%7COpen+Sans+Condensed:300' rel='stylesheet' type='text/css'>
-    
+
     <!--favicon -->
     <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon" />
     <!--TODO MISSING A TON OF FAVICONS -->
-    
+
         <title>dwyl - saves you time so you can do what you love</title>
-    
+
     <!--Google Analytics-->
-    <script type="text/javascript" src="https://mandrillapp.com/api/docs/js/mandrill.js"></script>
+    <!-- <script type="text/javascript" src="https://mandrillapp.com/api/docs/js/mandrill.js"></script> -->
```

Comment:
We don't need the mandrill api in the front-end because we've abstracted into a dwyl API method (*so we can keep our mandrill API secret...!*)